Sources/ResilientDecoding/Resilient.swift

```diff
@@ -62,10 +49,11 @@ public struct Resilient<Value: Decodable>: Decodable {
   func map<T>(transform: (Value) -> T) -> Resilient<T> {
     let value = transform(wrappedValue)
     #if DEBUG
-    return Resilient<T>(
-      value: value,
-      topLevelError: projectedValue.topLevelError,
-      errorsAtOffset: projectedValue.errorsAtOffset)
+    if let error = projectedValue.error {
+      return Resilient<T>(value, error: error)
+    } else {
+      return Resilient<T>(value)
+    }
```

Comment:
Do we need this `else` block since we're already doing the same below? https://github.com/airbnb/ResilientDecoding/pull/11/files#diff-c0df969f14dd120e657b56e5476ee8b0R58

Reply:
I think this is still true. We can probably simplify this by doing:

```swift
#if DEBUG
if let error = projectedValue.error {
  return Resilient<T>(value, error: error)
}
#endif 
return Resilient<T>(value
```